php/media/class-gallery.php

```diff
@@ -102,6 +101,13 @@ class Gallery {
 	 */
 	protected $config = array();
 
+	/**
+	 * Where the gallery script should be enqueued. false = header; true = footer.
+	 *
+	 * @var bool
+	 */
+	protected $gallery_in_footer = false;
```

Comment:
Could we keep this in the footer if we use the queue system which stores just the widget instances and their configs on the current page?

Reply:
Found a clean workaround for this!